src/patterns/gen-ai/aws-web-crawler/index.ts

```diff
@@ -353,7 +363,7 @@ export class WebCrawler extends BaseClass {
       replaceComputeEnvironment: true,
       updateTimeout: cdk.Duration.minutes(30),
       updateToLatestImageVersion: true,
-      vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_EGRESS },
+      vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_ISOLATED },
```

Comment:
Can you confirm that the crawler would still be able to access web pages with `PRIVATE_ISOLATED`?